src/components/post/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { useQuery } from '@tanstack/react-query';
+import { getPostFeedData } from './api';
+import { ListInfo } from './types';
+import { useRouter } from 'next/router';
+import { useRef } from 'react';
+
+function PostList() {
+  const { data, isLoading } = useQuery<ListInfo[]>(['postfeed'], getPostFeedData);
```

Comment:
```suggestion
  const { data: postList, isLoading } = useQuery<ListInfo[]>(['postfeed'], getPostFeedData);
```
p2; 이렇게 data 변수 네이밍 바꿔주면 코드 이해하기 더 좋을 것 같아요!